crates/uv-python/src/version_files.rs

```diff
@@ -32,7 +102,7 @@ pub async fn requests_from_version_file() -> Result<Option<Vec<PythonRequest>>,
 /// Read a [`PythonRequest`] from a version file, if present.
 ///
 /// Prefers `.python-version` then the first entry of `.python-versions`.
-/// If multiple Python versions are desired, use [`requests_from_version_files`] instead.
+/// If multiple Python versions are desired, use [`requests_from_version_file`] instead.
```

Comment:
Why this change?

Reply:
Is this supposed to be `requests_from_versions_file`?

Reply:
Yes, the docstring was wrong, `requests_from_version_files` doesn't exist.